mlflow/cli.py

```diff
@@ -194,14 +194,14 @@ def run(
 def _user_args_to_dict(arguments, argument_type="P"):
     user_dict = {}
     for arg in arguments:
-        split = arg.split("=")
+        split_index = arg.find("=")
         # Docker arguments such as `t` don't require a value -> set to True if specified
-        if len(split) == 1 and argument_type == "A":
-            name = split[0]
+        if split_index == -1 and argument_type == "A":
+            name = arg[:split_index]
```

Comment:
Should the name just be equal to `arg` here if there's no equals sign? It looks like we chop off the last character here, e.g:

```
>>> arg = "squid"
>>> split_index = arg.find("=")
>>> split_index
-1
>>> name = arg[:split_index]
>>> name
'squi'
```

Is that intentional?

Reply:
Thanks so much for catching this. I'd never want to chop the tail off a `squid`. Fixed based on @harupy 's suggestion!

Reply:
Added test coverage for `-A` arguments as well to `test_projects_cli`